src/main/resources/sqlline/SqlLine.properties

```diff
@@ -63,6 +63,8 @@ help-appconfig: Set custom application configuration class name
 help-set: List / set a sqlline variable
 help-rerun: Execute previous command from the history file
 help-reset: Reset a sqlline variable
+help-confirm: Additional step before executing desctructive commands - default: false
```

Comment:
Since we use a regex for the confirm pattern while the initial intention is for destructive commands this is not limited to them correct?